dataRetriever/constants.go

```diff
@@ -0,0 +1,15 @@
+package dataRetriever
+
+// TxPoolNumSendersToEvictInOneStep instructs tx pool eviction algorithm to remove this many senders when eviction takes place
+const TxPoolNumSendersToEvictInOneStep = uint32(100)
+
+// TxPoolALotOfTransactionsForASender instructs tx pool eviction algorithm to tag a sender with more transactions than this value
+// as a "sender with a lot of transactions"
+const TxPoolALotOfTransactionsForASender = uint32(500)
```

Comment:
MaximumNumOfTxsForASender?

Reply:
Renamed to `LargeNumOfTxsForASender`, since is not the actual maximum number of transactions that can be held for a sender.

During the eviction phase, there's a step that does this:

```
For senders with many transactions (> "LargeNumOfTxsForASender"), evict "NumTxsToEvictForASenderWithLargeNumOfTxs" transactions
```